### DISCUSSION TITLE
Dudas sobre Pokedex JS

### DISCUSSION THREAD

**Discussion by erickmatz9101** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#413] [Q&A]
Hola compañeros, tengo muchas dudas al momento de implementar la API, es la primera vez que ocupo APIS en un proyecto y se me esta complicando mas de lo que pensé, en un momento pensaba en poderlo implementar como lo hizo el MC pero no se que me esté fallando. NO busco que me hagan mi proyecto pero si me pudieran echar una mano se los agradecería. Gracias.

**Comment by Dfredude** [comment]
Cual es tu pregunta o duda? Tienes ser mas especifico. Saludos,

**Comment by doguedogue** [comment]
La idea es usar la API como en el vídeo, esa es la base, lo demás sería "enchularlo" con css y darle la estructura que te guste en HTML, el proyecto de ejemplo está aquí 

https://github.com/romarpla/FrontEnd-Mision/tree/main/04%20-%20JS/programas

Adelante 🚀

**Reply by erickmatz9101** [reply]
Si la idea es hacerlo asi y de hecho si lo estoy intentando asi pero no me sale.... Y vi el live y toda la cosa pero no se que esté haciendo mal

**Comment by carlos-gmx3** [comment]
Sube tu código para ver que falló
yo noté algo que no se explicó bien y es en esta linea
const url = `https://pokeapi.co/api/v2/pokemon/${poke}`;
y es que necesitas usar comillas invertidas ``
las comillas simples '' o dobles "" no van a funcionar si en tu cadena quieres integrar una variable

**Reply by erickmatz9101** [reply]
Claro mira te comparto lo poquito que llevo realizado, cuando yo veo en consola lo que mando a llamar me aparece el error 404
![Captura de pantalla (19)](https://user-images.githubusercontent.com/77296736/158512476-461939a1-40ab-4219-a349-e8cd4e799de6.png)
![Captura de pantalla (20)](https://user-images.githubusercontent.com/77296736/158512481-36120b2b-8c6e-4e3d-ad6a-77b2f4fd1cdc.png)

**Comment by carlos-gmx3** [comment]
Primer detalle
![imagen](https://user-images.githubusercontent.com/99064239/158514638-2ebf4eb7-83cf-4623-a166-5765a06da45b.png)
Que hace ese pokeNameInput solito? el getElement lo hiciste en las primeras lineas?
o solo q el detalle este en la función pokeImage
